spec/features/sessions/merchant_user_new_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "Logging in" do
+    before(:each) do
+        @regular_user = User.create(name: "Willy Wonka", address: "123 St", city: "Denver", state: "CO", zip: "12345", email: "chocolateguy1@gmail.com", password: "loco123", role: 0)
+        @merchant_1 = User.create(name: "Maude Sloggett", address: "17 Sun Rise St", city: "El Paso", state: "Illinois", zip: "56726", email: "M.Slogget@yahoo.com", password_digest: "Forever27", role: 1)
+    end
+
+    it "I see a field to enter my email address and password for regular user with logged in flash" do
+
+        visit "/"
+
+        click_on "Sign in"
+
+        expect(current_path).to eq('/login')
+
+        fill_in :email, with: @merchant_1.email
+        fill_in :password, with: @merchant_1.password
+        # how to test becrypt authentication
+
+        click_on "Login!"
+
+        expect(current_path).to eq('/')
```

Comment:
THis was eventually changed dont even look at this. Its trash.

Reply:
![](https://media3.giphy.com/media/yRNSxsl1rJEwU/giphy.gif?cid=5a38a5a2e0e9d3b01e8d7a0d5403dabda2e02ba4b2c2c3b5&rid=giphy.gif)